clearcut_detection_backend/update.py

```diff
@@ -1,79 +1,30 @@
-import os
-import shutil
+"""
+Updating mapbox tiles
+"""
 import traceback
-from os.path import join, splitext
 
-from django.core.mail.message import EmailMessage
+import django
 from django.conf import settings
+from django.core.mail import EmailMessage
 
-from clearcuts.geojson_save import save
-from model_call import raster_prediction
-from jp2_to_tiff_conversion import jp2_to_tiff
+django.setup()
 from sentinel_download import SentinelDownload
 from upload_to_mapbox import start_upload
-
-DATA_DIR = 'data'
-
-
-
-def download_tile(data_dir):
-    # TODO(flying_pi): peps_download.py must be refactored and added main enterpoint.
-    # os.system('python peps_download.py')
-    import peps_download
-    for file in os.listdir(data_dir):
-        if file.endswith('.json'):
-            os.remove(join(data_dir, file))
-        elif file.endswith('.zip'):
-            path = join(data_dir, splitext(file)[0])
-            os.system(f'unzip {path} -d {data_dir}')
-            os.remove(f'{path}.zip')
-            os.system(f'python prepare_tif.py -f {path}.SAFE')
-            shutil.rmtree(f'{path}.SAFE')
-
-
-def process_tile(data_dir):
-    for file in os.listdir(data_dir):
-        if file.endswith('.tif'):
-            tiff_path = join(data_dir, file)
-            a = 1/0
-            result_paths = raster_prediction(tiff_path)[0]
-            return result_paths["polygons"], result_paths["picture"]
-
-
-def init_db(data_dir):
-    files = sorted(os.listdir(data_dir))
-    for file in files:
-        save(os.path.join(data_dir, file), init_db=True)
-
-
-def update_db(data_dir):
-    # TODO download of satellite imagery can be moved to model api in order to reduce amount of times that it is
-    #  downloaded, but it also means that model instance will spend time downloading data(which is not correct behavior
-    #  for model instance with gpu)
-    download_tile(data_dir)
-    poly_path, image_path = process_tile(data_dir)
-    save(os.path.join(data_dir, poly_path), init_db=False)
-
+from jp2_to_tiff_conversion import jp2_to_tiff
 
 if __name__ == '__main__':
     try:
-        # TODO(flyingpi): Remove or rewrite django stuff after moving it to separate service.
-        import django
-        django.setup()
-        sd = SentinelDownload()
-        sd.process_download()
-        # TODO: need a proper schedule
-        # jp2_to_tiff()
-        # start_upload()
-
-        # older version
-        # update_db(DATA_DIR)
-    except Exception as e:
+        sentinel_downloader = SentinelDownload()
+        sentinel_downloader.process_download()
+        sentinel_downloader.executor.shutdown()
+        jp2_to_tiff()
+        start_upload().shutdown()
+    except Exception as error:
         EmailMessage(
             subject='Pep download issue',
             body=(
                 f'Deamon can not download peps. Issue information listed bellow: '
```

Comment:
change 'peps' to another name, since we are downloading Sentinel2 data and not using peps module anymore